content/docs/quickstart.mdx

```diff
@@ -22,74 +22,216 @@ keywords:
+Replace the following values:
 
-Run Pomerium Core with Docker containers in **under 5 minutes**.
+- **Line 8**: Replace `CLUSTER_TOKEN` with the cluster token generated for you in the onboarding config
+- **Line 15**: Replace `CLUSTER_STARTER_SUBDOMAIN` with your generated starter subdomain
 
-The Core quickstart uses Pomerium's [**Hosted Authenticate Service**](/docs/capabilities/hosted-authenticate-service), but you can also configure a [**self-hosted authenticate service**](/docs/capabilities/self-hosted-authenticate-service) to integrate with Pomerium.
+:::info Where's my **Starter Domain?**
 
-## Prerequisites
+You can find your **Starter Domain** at the top of the Zero Console in the navigation bar.
 
-[Docker] and [Docker Compose]
+:::
 
-## Configure Pomerium
 
-Create a `config.yaml` file in the root of your project.
+## Connect to Pomerium Zero
 
-Add the configuration below to `config.yaml`:
+Next, deploy Pomerium Zero with the following command:
 
-<ConfigDocker />
+```bash
+docker compose up -d
+```
 
-## Configure Docker
+This command will deploy a local cluster running Pomerium in a Docker container. If your cluster deploys successfully, the Pomerium Zero cloud will attempt to secure a connection to your cluster.
 
-Create a `docker-compose.yaml` file in the root of your project.
+If Pomerium Zero connects successfully, it will notify you in the onboarding screen.
 
-Add the configuration below to `docker-compose.yaml`:
+![The successful connection notification in the Pomerium Zero onboarding flow](./img/quickstart/zero-quickstart-connection.png)
 
-<DockerCompose />
+Once you've connected, select **Finish**.
 
-## Run Docker Compose
+:::info
+
+See the [Clusters Concept] page to learn more about clusters and how they work with the Pomerium Zero cloud.
+
+:::
+
+## Connect to upstream services
+
+In the Zero Console, we've provisioned two starter routes, **SSH** and **Verify**, that come with their own DNS records and TLS certificates:
+
+![Viewing the two starter routes](./img/quickstart/pz-quickstart-routes.png)
+
+### Connect to SSH
+
+In a terminal, enter the following command and replace `CLUSTER_STARTER_DOMAIN` with your own:
 
 ```bash
-docker compose up
+pomerium-cli tcp ssh.<CLUSTER_STARTER_DOMAIN>.app:22
+```
+
+A successful request results in a response similar to the one below:
+
+```shell-session
+$ pomerium-cli tcp ssh.dominant-cowbird-8176.pomerium.app:22
+2024/03/25 16:18:45 listening on 127.0.0.1:52541
 ```
 
-Access the **verify route** you built in your policy: `https://verify.localhost.pomerium.io`
+:::note
 
-If you get a self-signed certificate warning, see [Handle Self-Signed Certificate Warning](/docs/internals/troubleshooting#handle-self-signed-certificate-warning) to bypass it.
+If you haven't, install [**Pomerium CLI**](/docs/deploy/clients/pomerium-cli) with any of the following methods:
 
-You should be redirected to the **verify** service. You'll see a page like this:
+<Tabs>
```

Comment:
I wonder if it makes more sense to link out to a different page for the pomerium-cli installation instructions... it seems a hassle to keep duplicate copies of these instructions in sync.

Reply:
We can [import Markdown](https://docusaurus.io/docs/markdown-features/react#importing-markdown) in Docusaurus. This would make it so we have one source of truth that we can reuse and maintain.